docs/spec_draft.md

```diff
@@ -1420,6 +1421,79 @@ Produces an `output` tensor from a constant `value`.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.convert
+
+### Semantics
+
+Performs an element-wise conversion of values similar to an element-wise
+`static_cast` in C++ from one element type to another on `operand` tensor and
+produces a `result` tensor.
+
+For conversions involving **integer-to-integer**, if there is an unsigned/signed
+overflow, the result is implementation-defined and one of the following:
+  * mathematical result modulo $2^n$, where n is the bit width of the result,
+    for unsigned overflow. For signed integer overflow, wraps the result around
+    the representable range $[-2^{n-1},\ 2^{n-1} - 1]$.
+  * saturation to $2^{n-1} - 1$ (or $-2^{n-1}$) for signed overflow and
+    saturation to $2^n - 1$ (or $0$) for unsigned overflow.
+
+For conversions involving **floating-point-to-floating-point** or
+**integer-to-floating-point**, if the source value can be exactly represented in
+the destination type, the result value is that exact representation. If the
+source value is between two adjacent destination values, the result is
+implementation-defined. Otherwise, the behavior is undefined.
```

Comment:
How about we formulate this less ambitiously for now and leave figuring out the details (which way the rounding goes, is the "otherwise" case IB or UB) to #223?

```
For conversions involving **floating-point-to-floating-point** or
**integer-to-floating-point**, if the source value can be exactly represented in
the destination type, the result value is that exact representation. Otherwise,
the behavior is TBD.
```